test/Feature/SingularSpec.hs

```diff
@@ -100,25 +100,13 @@ spec =
       it "raises an error when attempting to create multiple entities" $ do
         p <- request methodPost
           "/addresses"
-          [("Prefer", "return=representation"), singular]
+          [singular]
           [json| [ { id: 200, address: "xxx" }, { id: 201, address: "yyy" } ] |]
         liftIO $ simpleStatus p `shouldBe` notAcceptable406
 
-        -- the rows should not exist, either
-        get "/addresses?id=eq.200" `shouldRespondWith` "[]"
-
-      it "return=minimal allows request to create multiple elements" $
-        request methodPost "/addresses"
-          [("Prefer", "return=minimal"), singular]
```

Comment:
Instead of deleting this test, perhaps we can rephrase it as `singular behavior holds priority over return=minimal` and confirm it returns 406.

Reply:
Hmm.. I'm hesitating about this part now, I think it merits more discussion. See my above comment.

Reply:
I think that expectation to have success code on creation multiple objects when you obviously set singular entity header is oddly and in real life no projects do this.